src/allotropy/allotrope/schema_mappers/adm/flow_cytometry/benchling/_2025/_03/flow_cytometry.py

```diff
@@ -0,0 +1,341 @@
+    CompensationMatrixAggregateDocument,
+    CompensationMatrixDocumentItem,
+    DataProcessingDocument,
+    DataRegionAggregateDocument,
+    DataRegionDocumentItem,
+    DataSystemDocument,
+    DeviceControlAggregateDocument,
+    DeviceControlDocumentItem,
+    DeviceSystemDocument,
+    FlowCytometryAggregateDocument,
+    FlowCytometryDocumentItem,
+    MatrixAggregateDocument,
+    MatrixDocumentItem,
+    MeasurementAggregateDocument,
+    MeasurementDocumentItem,
+    Model,
+    PopulationAggregateDocumentItem,
+    PopulationDocumentItem,
+    ProcessedDataAggregateDocument,
+    ProcessedDataDocumentItem,
+    SampleDocument,
+    StatisticsAggregateDocument,
+    StatisticsDocumentItem,
+    VertexAggregateDocument,
+    VertexDocumentItem,
+)
+from allotropy.allotrope.models.shared.definitions.custom import (
+    TQuantityValueMilliAbsorbanceUnit,
+    TQuantityValueRelativeFluorescenceUnit,
+    TQuantityValueUnitless,
+)
+from allotropy.allotrope.schema_mappers.schema_mapper import SchemaMapper
+from allotropy.constants import ASM_CONVERTER_VERSION
+from allotropy.exceptions import AllotropeConversionError
+
+
+@dataclass(frozen=True)
+class Vertex:
+    x_coordinate: float
+    y_coordinate: float
+    unit: TQuantityValueMilliAbsorbanceUnit | (TQuantityValueRelativeFluorescenceUnit)
+
+
+@dataclass(frozen=True)
+class DataRegion:
+    region_data_identifier: str | None
+    region_data_type: str | None
+    parent_data_region_identifier: str | None = None
+    x_coordinate_dimension_identifier: str | None = None
+    y_coordinate_dimension_identifier: str | None = None
+    vertices: list[Vertex] | None = None
+
+
+@dataclass(frozen=True)
+class CompensationMatrix:
+    dimension_identifier: str | None
+    compensation_value: float | None
+
+
+@dataclass(frozen=True)
+class CompensationMatrixGroup:
+    dimension_identifier: str | None
+    compensation_matrices: list[CompensationMatrix] | None = None
+
+
+@dataclass(frozen=True)
+class Statistic:
+    dimension_identifier: str
+    statistical_feature: str
+
+
+@dataclass(frozen=True)
+class Population:
+    population_identifier: str
+    written_name: str | None = None
+    data_region_identifier: str | None = None
+    parent_population_identifier: str | None = None
+    sub_populations: list[Population] | None = None
+    statistics: list[Statistic] | None = None
+
+
+@dataclass(frozen=True)
+class Measurement:
+    measurement_identifier: str
+    sample_identifier: str
+    device_type: str
+
+    location_identifier: str | None = None
+    well_plate_identifier: str | None = None
+    written_name: str | None = None
+    method_version: str | None = None
+    data_processing_time: str | None = None
+    processed_data_identifier: str | None = None
+    populations: list[Population] | None = None
+    data_regions: list[DataRegion] | None = None
+
+    sample_custom_doc: dict[str, Any] | None = None
```

Comment:
```suggestion
    sample_custom_info: dict[str, Any] | None = None
```
Please try to follow this naming convention. It is important that we are consistent.

Reply:
my bad, I keep forgetting